xonsh/environ.py

```diff
@@ -104,11 +105,13 @@ def to_debug(x):
     'LOADED_CONFIG': (is_bool, to_bool, bool_to_str),
     'LOADED_RC_FILES': (is_bool_seq, csv_to_bool_seq, bool_seq_to_csv),
     'MOUSE_SUPPORT': (is_bool, to_bool, bool_to_str),
+    'MULTILINE_PROMPT': (is_string_or_callable, ensure_string, ensure_string),
     re.compile('\w*PATH$'): (is_env_path, str_to_env_path, env_path_to_str),
     'PATHEXT': (is_env_path, str_to_env_path, env_path_to_str),
     'PRETTY_PRINT_RESULTS': (is_bool, to_bool, bool_to_str),
+    'PROMPT': (is_string_or_callable, ensure_string, ensure_string),
     'RAISE_SUBPROC_ERROR': (is_bool, to_bool, bool_to_str),
-    'RIGHT_PROMPT': (is_string, ensure_string, ensure_string),
+    'RIGHT_PROMPT': (is_string_or_callable, ensure_string, ensure_string),
```

Comment:
Thanks for adding these! I don't think that this will properly convert functions, but this is probably not a big deal for the moment.